src/BloomTests/web/EnhancedImageServerTests.cs

```diff
@@ -173,6 +173,64 @@ public void CanRetrieveContentOfFakeTempFile_ButOnlyUntilDisposed()
 			}
 		}
 
+		[Test]
+		public void CanRetrieveContentOfFakeTempFile_WhenFolderContainsAmpersand_ViaJavaScript()
+		{
+			using (var server = CreateImageServer())
+			{
+				var ampSubfolder = Path.Combine(_folder.Path, "Mom & Dad");
+				Directory.CreateDirectory(ampSubfolder);
+				var html = @"<html ><head><title>Mom &amp; Dad</title></head><body>here it is</body></html>";
+				var dom = new HtmlDom(html);
+				dom.BaseForRelativePaths = ampSubfolder.ToLocalhost();
+				string url;
+				using (var fakeTempFile = EnhancedImageServer.MakeSimulatedPageFileInBookFolder(dom, true))
```

Comment:
Well, there's also the 3rd optional argument on the PretendRequestInfo ctor...
I think the way I've redone it is clearer. The second test doesn't use either optional argument.